jenkins-pipelines/oss/rolling-upgrade/rolling-upgrade-with-enable-tablets.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
```

Comment:
is this file equivalent to jenkins-pipelines/oss/no_tablets/rolling-upgrade-activate-tablets.jenkinsfile ?
i guess it is not relevant anymore under `no_tablets` directory (my miss) so should be removed.

Reply:
If it's a file  you created, and you think it should be removed, please open a PR for that

Reply:
No. it actually uses a different test and different configuration.